example/centos/centos-9-ppc64le-tar.yaml

```diff
@@ -9,7 +10,7 @@ otk.define:
   packages:
     build:
       otk.external.osbuild-gen-depsolve-dnf4:
-        architecture: s390x
```

Comment:
This was a copy/paste error before, right? I wonder why we did not spot this, I guess because the only arch specific piece in the dnf resolve is the kernel version, probably worth looking into how we can test for this in a more reliable way (i.e. tar has no kernel so we did not notice)

Reply:
Yes this was an error before! It seems like something that maybe the depsolve mocking could catch if we include (part of) the depsolve request?

Reply:
I opened https://github.com/osbuild/images/pull/968 for this now